app/src/main/java/com/fady/brain_power_task/adapter/AthletesAdapter.kt

```diff
@@ -1,24 +1,37 @@
-package com.fady.brain_power_task
+package com.fady.brain_power_task.adapter
 
 import androidx.recyclerview.widget.RecyclerView
 import android.view.LayoutInflater
-import android.view.View
 import android.view.ViewGroup
-import androidx.recyclerview.widget.AsyncListDiffer
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
+import com.bumptech.glide.Glide
+import com.fady.brain_power_task.R
 import com.fady.brain_power_task.databinding.AthleteItemViewBinding
 import com.fady.brain_power_task.model.Athlete
 
-class AthletesAdapter(private val interaction: (athlete: Athlete, pos: Int) -> Unit?) :
-    ListAdapter<Athlete, AthletesAdapter.AthletesViewHolder>(DiffCallback()) {
+class AthletesAdapter(private val interaction: (athlete: Athlete, pos: Int) -> Unit) :
+    ListAdapter<Athlete, AthletesAdapter.AthletesViewHolder>(
+        DiffCallback()
+    ) {
+    class DiffCallback : DiffUtil.ItemCallback<Athlete>() {
+        override fun areItemsTheSame(oldItem: Athlete, newItem: Athlete): Boolean {
+            return oldItem == newItem
+        }
 
+        override fun areContentsTheSame(oldItem: Athlete, newItem: Athlete): Boolean {
+            return oldItem.name == newItem.name
+        }
+
+    }
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): AthletesViewHolder {
         val itemView = AthleteItemViewBinding.inflate(
             LayoutInflater.from(parent.context), parent, false
         )
-        return AthletesViewHolder(itemView)
+        return AthletesViewHolder(
+            itemView
```

Comment:
Is your Android Studio automatically formatting this spacing or is this how you like spacing it? The format we follow is `return AthletesViewHolder(itemView)` in the same line, let me know if AS is automatically doing this so we can try to fix it.

Reply:
yes It's automatically